docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -51,6 +51,8 @@ The authentication process follows this pathway:
 - **Keep it secret; keep it safe**: Treat your API key like a password.
   Do not expose it in client-side code, public repositories, or other easily accessible locations.
 - **Rotate keys regularly**: Change your API keys periodically to reduce risks from potential leaks.
+  Keys must be rotated every at least every 90 days.
+- **Design your code for easy key updates**: Opt for key management practices that let you refresh your API keys without touching your code.
```

Comment:
Nit: instead of saying "without touching code" I'd probably just say to use our cloud ops APIs to do automation

Reply:
I'd also remove the word "easy" because key rotation is generally considered a hard problem

Reply:
@mattkim Reworded to "- **Design your code for easy key updates**: Use key management practices that retrieve your API keys without hard-coding them into your apps.
  This lets you restart your Workers to refresh your rotated keys without recompiling your code."

Reply:
Have I missed the point?

Reply:
Removing `easy`.

Reply:
@mattkim OK? 

```
- **Design your code for key updates**: Use key management practices that retrieve your API keys without hard-coding them into your apps.
  This lets you restart your Workers to refresh your rotated keys without recompiling your code.
```